### PR TITLE
[MAT-6009] Build CQL Statement Relevance (Used/Unused) Map

### PR DESCRIPTION
**Pull request by jkotanchik-SB**
## MADiE PR

Jira Ticket: [MAT-6009](https://jira.cms.gov/browse/MAT-6009)
(Optional) Related Tickets:

### Summary
- Correct spelling of Association
- Port over the statement relevance map from Bonnie that will mark defines as used (TRUE), unused (FALSE), or NA.

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included
* [x] No extemporaneous files are included (i.e Complied files or testing results)
* [x] This PR is into the **correct branch**.
* [x] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [x] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/util/CalculationTestHelpers.ts [hunk 1]
```diff
@@ -120,3 +120,200 @@ export function getExampleValueSet(): fhir4.ValueSet {
...
+  populationSet
+) {
+  // create a population relevance map where every population is true.
+  const populationRelevance = {};
+
+  if (!_.isEmpty(populationSet[0]["observations"])) {
+    populationRelevance["observation_values"] = true;
+  }
+
+  // TODO can we assume a single populationSet?
```
Comment (adongare) [inline src/util/CalculationTestHelpers.ts]: we can assume this is going to be called for each group. so yeah, one group at a time. no need to access with index or iterate over sets

**Review by adongare — DISMISSED** [review]

**Review by sb-prateekkeerthi — APPROVED** [review]
